frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx

```diff
@@ -346,6 +363,14 @@ export const ProjectAccessAssign = ({
                                 disableCloseOnSelect
                                 disabled={edit}
                                 value={selectedOptions}
+                                getOptionDisabled={option => {
+                                    if (option.type === ENTITY_TYPE.GROUP) {
+                                        const optionGroup =
+                                            option.entity as IGroup;
+                                        return !!optionGroup.rootRole;
```

Comment:
Again, just a matter of preference, but maybe it's more readable and consistent with the rest of the codebase.
```suggestion
                                        return Boolean(optionGroup.rootRole);
```

Reply:
Ah! I asked in a previous comment. Yes makes sense, I'll accept both of these